proto/viam/app/data/v1/data.proto

```diff
@@ -173,3 +185,54 @@ message DeleteBinaryDataByIDsRequest {
 message DeleteBinaryDataByIDsResponse {
   uint64 deleted_count = 1;
 }
+
+// Annotation specifies the ways in which we can associate annotations with data, e.g. with a text tag or image bounding box
+message Annotation {
+  string tag = 1;
+}
+
+// AddAnnotationsToBinaryDataByFileIDsRequest requests adding annotations to the files when file IDs are provided
```

Comment:
Might be useful to specify here that all of the `annotations` are applied to each file, since both fields are repeated and could otherwise be interpreted as 1:1 mapping between file_id and annotation

Reply:
done!